pkg/mdns/routes.go

```diff
@@ -127,11 +127,11 @@ func (c *MicroShiftmDNSController) deletedRoute(obj interface{}) {
 
 func (c *MicroShiftmDNSController) exposeHost(host string) {
 	if !strings.HasSuffix(host, server.DefaultmDNSTLD) {
-		klog.V(2).InfoS("mDNS ignoring host without mDNS suffix", "host", host)
+		klog.V(2).Infof("mDNS ignoring host %q without mDNS suffix", host)
 		return
 	}
 
-	klog.Infof("mDNS: route found for", "host", host, "ips", c.myIPs)
+	klog.Infof("mDNS: route found for host %q on ips %q", host, c.myIPs)
```

Comment:
We may want to be consistent about mDNS messages prefix: "mDNS: ..." or "mDNS ..."

Reply:
Done.